docs/snippets/CMakeLists.txt

```diff
@@ -77,7 +83,17 @@ list(REMOVE_ITEM SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/dldt_optimization_guide2.c
 # create a static library
 
 add_library(${TARGET_NAME} STATIC ${SOURCES})
-target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../../src/core/template_extension/new/")
+target_include_directories(${TARGET_NAME} PRIVATE "${OpenVINO_SOURCE_DIR}/src/inference/include/ie"
+                                                  "${OpenVINO_SOURCE_DIR}/src/inference/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/inference/dev_api"
+                                                  "${OpenVINO_SOURCE_DIR}/src/core/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/bindings/c/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/common/transformations/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/common/preprocessing"
+                                                  "${OpenVINO_SOURCE_DIR}/src/common/util/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/common/low_precision_transformations/include"
+                                                  "${OpenVINO_SOURCE_DIR}/src/frontends/common/include"
```

Comment:
do we really need LP, utils and transformations include in code snippets?

Reply:
Yes we use it here